Do not work with Python prior to 3.6 because fstring is used

Hello.
I'm getting error using this library with python 3.5 due to the usage of fstring.
It will be better to either not use fstring or clearly documentate that python 3.6+ is required.

```
../../../virtualenv/python3.5.7/lib/python3.5/site-packages/sqlakeyset/paging.py:6: in <module>
    from .columns import parse_clause, find_order_key
E     File "/home/travis/virtualenv/python3.5.7/lib/python3.5/site-packages/sqlakeyset/columns.py", line 43
E       f"Ordering by nullable column {x} can cause rows to be "
E                                                              ^
E   SyntaxError: invalid syntax
```

--- a/sqlakeyset/columns.py
+++ b/sqlakeyset/columns.py
@@ -40,9 +40,9 @@ def _warn_if_nullable(x):
     try:
         if x.nullable or x.property.columns[0].nullable:
             warn(
-                f"Ordering by nullable column {x} can cause rows to be "
+                "Ordering by nullable column {} can cause rows to be "
                 "incorrectly omitted from the results. "
-                "See the sqlakeyset README for more details."
+                "See the sqlakeyset README for more details.".format(x)
             )
     except (AttributeError, IndexError, KeyError):
         pass

